include/cutlass/cutlass.h

```diff
@@ -115,205 +115,254 @@ static const int NumThreadsPerQuadPair = NumThreadsPerQuad * 2;
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
 CUTLASS_HOST_DEVICE uint ThreadIdxX() {
-#if defined(CUTLASS_ENABLE_SYCL)
+#if defined(__CUDA_ARCH__)
+  return threadIdx.x;
+#elif defined(__SYCL_Device_ONLY__)
```

Comment:
I believe the capitalization of `DEVICE` in `__SYCL_**Device**_ONLY__` is missing, should've been, `__SYCL_DEVICE_ONLY__`, 

I will open a PR ton fix this 👍🏻